### ISSUE TITLE
Randomly can't contact docker registry

### ISSUE THREAD

**Issue by dberesford** [nearform/nscale#88]
Have hit this a few times, let me know what more info you need:

```{"name":"nscale-util","hostname":"Damians-MacBook-Pro.local","pid":10376,"level":30,"msg":"executor: docker push localhost:8011/cp_deployer/postgres-postgres_9.4, in: /Users/dberesford2/work/coderdojo/cp_deployer/.","time":"2015-05-14T08:21:07.245Z","v":0}
{"name":"nfd-kernel","hostname":"Damians-MacBook-Pro.local","pid":10376,"level":30,"msg":"stderr: 2015/05/14 09:21:07 Error: v1 ping attempt failed with error: Get http://localhost:8011/v1/_ping: dial tcp 127.0.0.1:8011: connection refused\n","time":"2015-05-14T08:21:07.410Z","v":0}
{"name":"nscale-util","hostname":"Damians-MacBook-Pro.local","pid":10376,"level":50,"msg":"executor: docker push localhost:8011/cp_deployer/postgres-postgres_9.4, in: /Users/dberesford2/work/coderdojo/cp_deployer/., stderr: 2015/05/14 09:21:07 Error: v1 ping attempt failed with error: Get http://localhost:8011/v1/_ping: dial tcp 127.0.0.1:8011: connection refused\n","time":"2015-05-14T08:21:07.410Z","v":0}
{"name":"nscale-util","hostname":"Damians-MacBook-Pro.local","pid":10376,"level":50,"msg":"executor: docker push localhost:8011/cp_deployer/postgres-postgres_9.4, in: /Users/dberesford2/work/coderdojo/cp_deployer/., FAILED!: 1","time":"2015-05-14T08:21:07.411Z","v":0}
{"name":"nfd-kernel","hostname":"Damians-MacBook-Pro.local","pid":10376,"module":"docker","level":50,"err":{"message":"command failed","name":"Error","stack":"Error: command failed\n    at ChildProcess.<anonymous> (/Users/dberesford2/.nvm/v0.10.38/lib/node_modules/nscale/node_modules/nscale-kernel/node_modules/nscale-util/lib/executor.js:62:17)\n    at ChildProcess.emit (events.js:117:20)\n    at maybeClose (child_process.js:766:16)\n    at Socket.<anonymous> (child_process.js:979:11)\n    at Socket.emit (events.js:95:17)\n    at Pipe.close (net.js:466:12)","code":1},"msg":"command failed","time":"2015-05-14T08:21:07.413Z","v":0}
{"name":"nfd-kernel","hostname":"Damians-MacBook-Pro.local","pid":10376,"level":50,"err":{"message":"command failed","name":"Error","stack":"Error: command failed\n    at ChildProcess.<anonymous> (/Users/dberesford2/.nvm/v0.10.38/lib/node_modules/nscale/node_modules/nscale-kernel/node_modules/nscale-util/lib/executor.js:62:17)\n    at ChildProcess.emit (events.js:117:20)\n    at maybeClose (child_process.js:766:16)\n    at Socket.<anonymous> (child_process.js:979:11)\n    at Socket.emit (events.js:95:17)\n    at Pipe.close (net.js:466:12)","code":1},"msg":"command failed","time":"2015-05-14T08:21:07.413Z","v":0}
{"name":"nfd-kernel","hostname":"Damians-MacBook-Pro.local","pid":10376,"level":30,"msg":"stdout: Error: command failed","time":"2015-05-14T08:21:07.413Z","v":0}
{"name":"nfd-kernel","hostname":"Damians-MacBook-Pro.local","pid":10376,"level":30,"msg":"stdout: Error: command failed","time":"2015-05-14T08:21:07.414Z","v":0}
{"name":"nfd-kernel","hostname":"Damians-MacBook-Pro.local","pid":10376,"level":50,"err":{"message":"command failed","name":"Error","stack":"Error: command failed\n    at ChildProcess.<anonymous> (/Users/dberesford2/.nvm/v0.10.38/lib/node_modules/nscale/node_modules/nscale-kernel/node_modules/nscale-util/lib/executor.js:62:17)\n    at ChildProcess.emit (events.js:117:20)\n    at maybeClose (child_process.js:766:16)\n    at Socket.<anonymous> (child_process.js:979:11)\n    at Socket.emit (events.js:95:17)\n    at Pipe.close (net.js:466:12)","code":1},"msg":"command failed","time":"2015-05-14T08:21:07.414Z","v":0}
{"name":"nfd-kernel","hostname":"Damians-MacBook-Pro.local","pid":10376,"level":30,"msg":"stderr: Error: command failed","time":"2015-05-14T08:21:07.414Z","v":0}
{"name":"nfd-kernel","hostname":"Damians-MacBook-Pro.local","pid":10376,"level":30,"msg":"rsp: build all containers g214e247-09ed-4e73-b5b8-6bd89d8d765c","time":"2015-05-14T08:21:07.414Z","v":0}
{"name":"nfd-kernel","hostname":"Damians-MacBook-Pro.local","pid":10376,"level":30,"err":{"message":"command failed","name":"Error","stack":"Error: command failed\n    at ChildProcess.<anonymous> (/Users/dberesford2/.nvm/v0.10.38/lib/node_modules/nscale/node_modules/nscale-kernel/node_modules/nscale-util/lib/executor.js:62:17)\n    at ChildProcess.emit (events.js:117:20)\n    at maybeClose (child_process.js:766:16)\n    at Socket.<anonymous> (child_process.js:979:11)\n    at Socket.emit (events.js:95:17)\n    at Pipe.close (net.js:466:12)","code":1},"msg":"command failed","time":"2015-05-14T08:21:07.414Z","v":0}
{"name":"nfd-kernel","hostname":"Damians-MacBook-Pro.local","pid":10376,"level":30,"msg":"stderr: command failed","time":"2015-05-14T08:21:07.414Z","v":0}```

**Comment by mcollina** [comment]
Me too. I am not exactly sure what the problem is, as it's spotty and hard to reproduce. I will have a look.

**Comment by mcollina** [comment]
I have just released 0.16.1, which should solve the problem (hopefully).
